lingua_franca/lang/parse_en.py

```diff
@@ -1193,8 +1354,8 @@ def date_found():
                     if (
                             int(strNum) > 100 and
                             (
-                                wordPrev == "o" or
-                                wordPrev == "oh"
+                                    wordPrev == "o" or
```

Comment:
Could be slightly simplified to: `wordPrev in ("o", "oh")`

Reply:
this is just autopep8, i didnt touch these lines